projects/plugins/jetpack/extensions/blocks/blogging-prompt/editor.js

```diff
@@ -38,7 +38,7 @@ async function insertTemplate( promptId ) {
 
 	const { insertBlocks } = dispatch( 'core/block-editor' );
 	const bloggingPromptBlocks = [
-		createBlock( `jetpack/${ name }`, { promptFetched: false, promptId, tagsAdded: true } ),
+		createBlock( 'jetpack/blogging-prompt', { promptFetched: false, promptId, tagsAdded: true } ),
```

Comment:
I considered importing name from the block.json file, but really it seems unnecessary. The name should be stable at this point and im seeing other blocks defined in this area similarly just hardcode the name.